qfieldsync/core/cloud_api.py

```diff
@@ -133,9 +201,21 @@ def __init__(self, parent=None) -> None:
         self.is_login_active = False
 
         url = self.preferences.value("qfieldCloudServerUrl")
+
         # we should always use the QgsNetworkAccessManager instance, otherwise ssl handling is impossible
         self._nam = QgsNetworkAccessManager.instance()
 
+        self.auth_method = CloudAuthMethod(
+            self.preferences.value("qfieldCloudAuthMethod")
+        )
+        pref_auth_config_id = self.preferences.value("qfieldCloudAuthcfg")
+        if pref_auth_config_id and pref_auth_config_id != "":
```

Comment:
This can be simplified to just 
```python
if pref_auth_config_id:
```

`if value` in Python will evaluate to `False` for `None`, `False`, `0`, `""`, `[]`, `{}`, ... and to `True` otherwise. It's usually considered more idiomatic Python, except in cases where you actually need to distinguish `None` from the empty string, for example.